src/routes/Root.tsx

```diff
@@ -0,0 +1,11 @@
+import { NavBar } from '../components/';
+import { Outlet } from 'react-router-dom';
+
+const Root = () => (
+	<main>
+		<Outlet />
+		<NavBar />
+	</main>
```

Comment:
Semantically, the `<nav>` tag would be outside of the `<main>` tag since it controls the contents of `<main>`. You can use a `<>` (fragment) here instead since the root is already a div.